### PR TITLE
Withdraw ECIP-1067 - Standardize OS Level Management of Signature Handling ("invoke-signer")

### PR DESCRIPTION
**Pull request by zmitton**

### CODE REVIEW

File: _specs/ecip-1067.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 lang: en
 ecip: 1067
 title: Standardize OS Level Management of Signature Handling ("invoke-signer")
-status: Draft
+status: REJECTED
```
Comment (soc1c) [inline _specs/ecip-1067.md]:
```suggestion
status: Withdrawn
```

File: _specs/ecip-1067.md [hunk 2]
```diff
@@ -93,16 +93,8 @@ Most users will have a favorite mobile signer and enjoy a consistent experience
...
-
-[ECIP 1037 -- Simple Non-Interactive URI Scheme](https://github.com/ethereumproject/ECIPs/pull/81)
-
-[EIP 67 -- Standard URI scheme with metadata value and byte code](https://github.com/ethereum/EIPs/issues/67)
-
-[EIP 681 -- Payment request URL specification](https://github.com/ethereum/EIPs/pull/681)
+---
 
-[EIP 831 -- Standard URL Format](https://github.com/ethereum/EIPs/pull/831)
+### Rejection Notice:
```
Comment (BelfordZ) [inline _specs/ecip-1067.md]:
```suggestion
### Withdrawal Notice:
```

**Comment by bobsummerwill** [comment]
So the author is Withdrawing this ECIP.
Please can we get this merged?  @meowsbits @soc1c @sorpaas @BelfordZ @YazzyYaz?

**Comment by bobsummerwill** [comment]
Thanks, @soc1c

**Pull request merged**